routes/index.js

```diff
@@ -311,11 +302,15 @@ var routes = {
       location = "thunderbird/" + location;
     }
     var frequency = request.params.frequency || 'single';
+    var options = {
+      recurring: frequency === 'monthly',
+      accountType: request.params.accountType
```

Comment:
`accountType` is technically going to be undefined until we add the micro/macro stuff. I meant to remove this line in this patch, but it's probably harmless.